Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM postgres:10.2-alpine
+FROM postgres:10.4-alpine
 
 MAINTAINER erik@timescale.com
 
-ENV PG_MAJOR 10.2
-ENV TIMESCALEDB_VERSION 0.9.1
-ENV PG_PROMETHEUS_VERSION 0.0.1
+ENV PG_MAJOR 10.4
+ENV TIMESCALEDB_VERSION 0.10.0
```

Comment:
Looks good, but wondering if we should wait with merging until 0.10.1 is out since it will have the session memory leak fix.